lib/posix/pthread.c

```diff
@@ -18,6 +18,8 @@
 #include <zephyr/sys/slist.h>
 #include <zephyr/sys/util.h>
 
+pid_t getpid(void);
```

Comment:
@jaiiarora 
Is there a reason that this is redeclared here? Was it not possible to get the prototype via
```cpp
#include <include/zephyr/posix/unistd.h>
```
?

Reply:
Will remove it.